app/javascript/react/components/Modals/TimelapseModal/TimelapseComponent.tsx

```diff
@@ -43,10 +43,10 @@ const TimelapseComponent: React.FC<
   const resetTimelapse = useCallback(() => {
     setCurrentStep(0);
     dispatch(setCurrentTimestamp(""));
-  }, []);
+  }, [dispatch]);
 
   const filteredTimestamps = useMemo(() => {
-    const now = moment.utc();
+    const now = moment();
```

Comment:
Don't we use utc on the backend? Shouldn't we convert user's local time to utc then? Did the previous fitleredTImestamps solution work only for our timezone but not for Michael's?